lldb/test/API/lang/cpp/std-function-recognizer/TestStdFunctionRecognizer.py

```diff
@@ -99,11 +99,8 @@ def test_api(self):
         (target, process, thread, bkpt) = lldbutil.run_to_source_breakpoint(
             self, "// break here", lldb.SBFileSpec("main.cpp")
         )
-        frame = thread.GetSelectedFrame()
         num_hidden = 0
-        for i in range(1, thread.GetNumFrames()):
-            thread.SetSelectedFrame(i)
-            frame = thread.GetSelectedFrame()
```

Comment:
this code didn't need to set the selected frame, it only needs to inspect whether a frame is hidden (which can be done without selecting it).